src/main/java/com/exasol/adapter/installer/VirtualSchemaInstallerConstants.java

```diff
@@ -3,83 +3,63 @@
 import java.util.Arrays;
 import java.util.stream.Collectors;
 
+import com.exasol.adapter.installer.dialect.Dialect;
+
+/**
+ * Virtual Schema installer constants.
+ */
 public class VirtualSchemaInstallerConstants {
```

Comment:
Seems to me that most of these constants belong to the `UserInput` class. I would suggest to move them where they are used

Reply:
I have already moved all constants that only belonged to one place